contracts/0.8.25/vaults/dashboard/Dashboard.sol

```diff
@@ -298,6 +312,7 @@ contract Dashboard is NodeOperatorFee {
     /**
      * @notice Accepts the ownership over the StakingVault and connects to VaultHub. Can be called to reconnect
      *         to the hub after voluntaryDisconnect()
+     * @dev correct the settled growth before reconnecting in order to enable the fee accrual for the node operator
```

Comment:
we can probably revert if the fee accrual is stopped and detect via uint104 max value reading